tests/a-framework/CMakeLists.txt

```diff
@@ -10,14 +10,23 @@ SET(TEST_TARGET_DEBUG dummy.debug)
 DEAL_II_PICKUP_TESTS()
 
 #
-# Limit concurrency between the two parameter file tests:
+# Limit concurrency between the two parameter file tests.
 #
-FOREACH(_build ${DEAL_II_BUILD_TYPES})
-  STRING(TOLOWER ${_build} _build)
-  SET_TESTS_PROPERTIES(a-framework/parameter_file_2.${_build} PROPERTIES
-    DEPENDS a-framework/parameter_file_1.${_build}
-    )
-ENDFOREACH()
+# We only do this for CMake-3.4 or newer that has policy CMP0064 and
+# supports the IF(TEST ...) operator.
```

Comment:
why do you need this only for 3.4+ ? Can you leave a comment?

Reply:
The problem is that a CMake before version 3.4 doesn't have the functionality in question.